src/TilemapShader.ts

```diff
@@ -52,7 +52,8 @@ export class TilemapGeometry extends Geometry
 	        .addAttribute('aTextureCoord', buf, 0, false, 0, this.stride, 2 * 4)
 	        .addAttribute('aFrame', buf, 0, false, 0, this.stride, 4 * 4)
 	        .addAttribute('aAnim', buf, 0, false, 0, this.stride, 8 * 4)
-	        .addAttribute('aTextureId', buf, 0, false, 0, this.stride, 10 * 4);
+	        .addAttribute('aTextureId', buf, 0, false, 0, this.stride, 10 * 4)
+            .addAttribute('aAlpha', buf, 0, false, 0, this.stride, 11 * 4);
```

Comment:
Can you fix this indent?

Reply:
hmm, I dont think this is indented in the file. If you click to edit the file, the indent disappears